### ISSUE TITLE
Setting the code folding menu item-shortcuts fail (on Windows?)

### ISSUE THREAD

**Issue by petetnt** [adobe/brackets#11004]
Windows 8.1., Brackets - Release 1.3 development build 1.3.0-0 (and the release itself) (no extensions)

1. Open Brackets
2. Check Developer Tools
3. See two of these: `Error setting menu item shortcut: 1`

Threw a console.log before the [line that throws the error](https://github.com/adobe/brackets/blob/master/src/command/Menus.js#L818) and the key combinations are `ctrl+alt+[` and `ctrl+alt+]` which according to @redmunds `display-brackets-shortcuts` extension belong to `codefolding.collapse` and `codefolding.expand`.

**Comment by redmunds** [comment]
@petetnt Yes, I need to update my `display-brackets-shortcuts` extension to use another shortcut.

Does anything not work as expected? I have been seeing a similar error message on Brackets 1.2 (that I have not gotten around to reporting) with a shortcut override, but everything still works as expected.

**Comment by petetnt** [comment]
The menu items do work, but the shortcuts don't, but that's most likely because I am on a finnish keyboard layout and pressing CTRL+ALT+[ or ] is AFAIK impossible as [ and ] are behind Alt Gr

![image](https://cloud.githubusercontent.com/assets/7641760/7355599/997cb49a-ed2b-11e4-8106-49803d45e5aa.png)
(from http://frontype.com/keyboarding/540px-Computer-keyboard-Sweden.svg.png)

**Comment by prafulVaishnav** [comment]
Similar to #10970

**Comment by mihawk90** [comment]
I am receiving the same error (Release 1.3 Build 1.3.0-16022).

I noticed that I didn't receive any errors on my Brackets installation on my Laptop (both my PC and my Laptop are Win8.1 64bit), so I went ahead and renamed my brackets folder in %appdata% to reset my settings, after that I didn't receive any errors.

I assume this is some older setting that is conflicting with something in the new code, as my Laptop's installation is completely new (1.3 is the first version ever installed on it), but the version on my PC has just been updated since 0.9 or something, I don't even remember.

**Comment by ficristo** [comment]
Reloading without extensions now I see on the console three warning like this:
``Ctrl-Shift-` `` (View -> Toggle Panels), `Ctrl-Alt-[`, `Ctrl-Alt-]`
They all seems impossible shortcut on Windows.
